src/Discord.Net/WebSocket/Extensions/SocketClientExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+
+namespace Discord.WebSocket
+{
+    public static class SocketClientExtensions
+    {
+        public static IVoiceRegion GetVoiceRegion(this DiscordSocketClient client, string id)
+        {
+            var region = client.VoiceRegions.FirstOrDefault(r => r.Id == id);
+            return region;
+        }
+
+        public static IReadOnlyCollection<IVoiceRegion> GetVoiceRegions(this DiscordSocketClient client) =>
+            client.VoiceRegions;
+
+        public static IGuild GetGuild(this DiscordSocketClient client, ulong id) =>
+            client.DataStore.GetGuild(id);
+
+        public static GuildEmbed? GetGuildEmbed(this DiscordSocketClient client, ulong id)
+        {
+            var guild = client.DataStore.GetGuild(id);
+            if (guild != null)
+                return new GuildEmbed(guild.IsEmbeddable, guild.EmbedChannelId);
+            return null;
+        }
+
+        public static IReadOnlyCollection<IUserGuild> GetGuildSummaries(this DiscordSocketClient client) =>
```

Comment:
Unsure if we should expose this one, especially since it's the same as GetGuilds